pipelines-in-pipelines/config/100-namespace.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: tekton-pip-run
```

Comment:
Any special reason for running this in a different namespace by default?

Reply:
we run them in a separate namespace to make it clear what was created by Pipelines in Pipelines controller, and make it easy to delete all its resources as needed

Reply:
if/when the feature is promoted from experimental to top level, we can switch to `tekton-pipelines` namespace at that time

Reply:
/cc @ImJasonH